lib/cc/analyzer/engine.rb

```diff
@@ -92,8 +92,14 @@ def docker_run_command
       end
 
       def env_file
+        contents = "ENGINE_CONFIG=#{@config_json}"
+
+        if contents.size > 64 * 1024
+          raise EngineFailure, "Config for engine #{name} exceeds 64k character limit"
```

Comment:
Agreed. We could suggest excluding fewer files, since that is the main contributor to this size, but that doesn't seem like a good UX either.

If it helps, this will only be around until next week when we move to a different solution.